README.md

```diff
@@ -1,3 +1,94 @@
-# geomancer
+![Geomancer logo](https://i.imgur.com/Zj5Gcd6.jpg)
 
-Automated OSM Feature Engineering built on top of BigQuery
+---
+
+Geomancer is a geospatial feature engineering Python package for OpenStreetMap (OSM) data that are saved in BigQuery. This project is currently in pre-alpha and is under active development.
+
+
+# Dependencies
+
+- Python 3.6.0+
+- geoalchemy2
+- google-cloud-bigquery
+- loguru
+- numpy
+- pandas
+- pandas-gbq
+- pytz
+- sqlalchemy
+
+All of the core dependencies can be found in `requirements.in`.
+
+# Setup for Active Testers and Developers
+
+## Installation
+
+1. Clone and enter the repository.
+```
+git clone https://github.com/thinkingmachines/geomancer.git
+cd geomancer
+```
+
+2. Make a virtual environment, install pip-tools, and activate the virtual environment.
+
+Mac/Linux users:
+```
+make venv
+```
+
+Windows users:
+```
+python -m venv venv
+venv\Scripts\activate
+pip install pip-tools
+```
+
+3. Install the dependencies.
+
+Mac/Linux users:
+```
+make build
+make dev
+```
+
+Windows users:
+```
+pip install -r requirements.txt
+pip install -r requirements-dev.txt
+```
+
+Now you're all set! Go out and cast spells or even make a few of your own. 🧙‍♀️🧙‍♂️✨
```

Comment:
Use github emojis so that they can be viewed across browsers/machines: https://gist.github.com/roachhd/1f029bd4b50b8a524f3c

Reply:
Removed the mage emoji to avoid emoji compatibility issues.

Reply:
```suggestion
Now you're all set! Go out and cast spells or even make a few of your own. ✨✨✨
```